src/main/scala/no/ndla/draftapi/service/search/ArticleIndexService.scala

```diff
@@ -42,7 +42,8 @@ trait ArticleIndexService {
           keywordField("defaultTitle"),
           textField("authors") fielddata true,
           textField("articleType") analyzer "keyword",
-          textField("notes") fielddata true
+          textField("notes") fielddata true,
+          textField("users") fielddata true,
```

Comment:
Betyr vel enda mindre her enn i search-api, men tror vi vil bruke `keywordField` istedet :)

Reply:
Kan godt endre. Men trur egentlig ikkje det er behov for å søke etter brukere her.

Reply:
Ser den, men fielddata felter er dyre (https://www.elastic.co/guide/en/elasticsearch/reference/current/fielddata.html#_fielddata_is_disabled_on_literal_text_literal_fields_by_default)
Så tror vi burde begrense de :)